Obsidian/Serializer/Enums/DataType.cs

```diff
@@ -21,10 +21,14 @@ public enum DataType
         Slot,
         NbtTag,
         Position,
+        AbsolutePosition,
```

Comment:
Mind telling what's the difference between Position and AbsolutePosition?

Reply:
Absolute position uses the Position class but se/de-serializes by reading/writing the x, y and z separately while Position would usually mean you're getting/writing the values from/to a long

Reply:
Hmm, weird name choice.
Might be worth documenting this encoding on the enum value?

Reply:
Sounds good, lets do that